jobs/integr8ly/installation-pipeline.yaml

```diff
@@ -158,5 +158,8 @@
             // Clean up - remove jenkins slave that was created
         
             Node bastionSlave = Jenkins.instance.getNode(bastionLabel);
-            Jenkins.instance.removeNode(bastionSlave);
+            Boolean isIdle = bastionSlave.toComputer().isIdle();
```

Comment:
I don't think that this solves the problem. If `bastionSlave` is null, then you get NPE on this line (calling `toComputer()` method on null object). The correct order should be:
1) null check
2) if not null see whether the slave is idle
2) remove the slave if it is not null and idle

Reply:
ahh yes good catch. Thanks